### ISSUE TITLE
Method `load_auto_load_scripts` throws an error when trying to create autoloaded scene files.

### ISSUE THREAD

**Issue by ericdsw** [bitwes/Gut#106]
I noticed that when running my tests, the console was throwing the following error before running any test:

```
SCRIPT ERROR: load_auto_load_scripts: Invalid call. Nonexistent function 'new' in base 'PackedScene'.
   At: res://addons/gut/gut_cmdln.gd:463.
```

And looking at the code, there part where autoloads are created is only accounting for `.gd` files, not `.tscn`:

```gdscript
var obj = load(f.get_value('autoload', key).replace('*', '')).new()
```
What I would recommend is check whether the return value of the `load` method returns an instance of `PackedScene`, which will instance the object via the `instance()` method, or if it's just a regular script resource in which case the `new()` method will be used.

```gdscript
var obj = load(f.get_value('autoload', key).replace('*', ''))
if obj is PackedScene:
    obj = obj.instance()
else:
    obj = obj.new()
```

**Comment by bitwes** [comment]
I had a hunch, and it looks like that code isn't needed anymore.  I'm not sure how far back Godot changed their loading of singletons but it appears to pick them up when running a script now too.  Could you verify that?  If it's not needed I'll take it out of 6.7.0 which should be out soon.

**Comment by ericdsw** [comment]
I just checked and it appear that autoloads are indeed loaded when running scripts now, so the `load_autoload_scripts()` method is not necessary.

Just a quick recap of what I did to test this, in case I overlooked something:

1. Replace `load_autoload_scripts` in `gut_cmdln.gd` with an empty body (just the `pass` keyword)

```gdscript
func load_autoload_scripts():
    pass
```

2. Create simple autoloads with just one method and registered them to the project's `Autoload` tab

**TestAutoloadScene.gd**:
```gdscript
extends Node

func return_two():
    return 2
```

**TestAutoloadScript.gd**
```gdscript
extends Node

func return_one():
    return 1
```

<img width="1078" alt="Screen Shot 2019-03-11 at 12 26 54 PM" src="https://user-images.githubusercontent.com/3594423/54144307-6ca23c00-43f9-11e9-9b70-3034957a9048.png">

3. Finally, created the following test in `test/unit/test_SampleUnitTest.gd`

```gdscript
extends "res://addons/gut/test.gd"

func test_this_is_a_sample_unit_test():

    var result = TestAutoloadScript.return_one()
    var another_result = TestAutoloadScene.return_two()
    assert_eq(result, 1)
    assert_eq(another_result, 2)
```

Running gut via the terminal works as expected:

<img width="639" alt="Screen Shot 2019-03-11 at 12 33 20 PM" src="https://user-images.githubusercontent.com/3594423/54144547-ee926500-43f9-11e9-9d88-a6d0d7023b98.png">

To clarify, I'm using Godot v3.0.6 stable.
